dist/redux-modules.cjs.js

```diff
@@ -1228,22 +1228,6 @@ function get(object, path, defaultValue) {
 
 var get_1 = get;
 
-/**
```

Comment:
Not clear to me why this was deleted? If it's just cleanup can you put in a separate commit please

Reply:
That's the Babel transpiled dist file, so you can ignore it.

Reply:
Ah, nice, no worries. Yeah we should do that someday